docs/troubleshooting-guide/create-har-file.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Creating a HAR file
+---
+
+# Creating a HAR file for troubleshooting
+
+HAR (short for _HTTP Archive_) files record network requests generated by your browser. HAR files can be helpful when troubleshooting issues with Metabase performance.
+
+> HAR files can contain sensitive information that you submit during the recording, like passwords and API keys. Before sharing the HAR file with anyone, review the file contents in a text editor.
+
+## Create a HAR file in Chrome
```

Comment:
I get why the headings are repetitive, but it's not great that the distinguishing info (the browser name) trails each heading. Might be helpful to have a small TOC in the first section that lists the name of the browser so people can skim and click to jump to the relevant section. E.g.,

- Chrome
- Edge
- Firefox
- Safari

Also, might want to sort the browsers alphabetically.